Code/src/main/java/nl/utwente/viskell/haskell/env/FunctionInfo.java

```diff
@@ -16,20 +24,38 @@
     public FunctionInfo(String name, Type signature) {
         this.name = name;
         this.signature = signature;
+        this.requiredBlocks = Collections.EMPTY_LIST;
```

Comment:
Constructors can call other constructors in Java, so this should also work:

    this(name, signature, ImmutableList.of())